lib/util/player_configuration.js

```diff
@@ -180,6 +181,7 @@ shaka.util.PlayerConfiguration = class {
       observeQualityChanges: false,
       maxDisabledTime: 30,
       parsePrftBox: false,
+      codecSwitchingStrategy: shaka.config.CodecSwitchingStrategy.DISABLED,
```

Comment:
I think the default behavior should be SMOOTH since it is a feature that we want to have enabled by default.